tick/survival/sccs/batch_convolution_sccs.py

```diff
@@ -0,0 +1,177 @@
+from tick.survival.sccs.conv import *
```

Comment:
filename: convolution -> convolutional ? (To be consistent with */sccs/convolutional_sccs.py